inst/eDISH_app/modules/renderSettings/renderSettingsUI.R

```diff
@@ -5,11 +5,31 @@ renderSettingsUI <- function(id){
   
   tagList(
     verticalLayout(
-      
       fluidRow(
-        column(6,
+        column(4,
```

Comment:
Will be needed soon, but I'm thinking we should comment this out for the v0.9.0 release, and then add it back in once we have a 2nd chart.